_includes/layout/navigation.html

```diff
@@ -132,10 +132,10 @@
     <div class="nav-popup" role="menu" aria-hidden="true">
       <ul>
         {% if page.guide_version != "2.0" %}
-        <li><a  data-proofer-ignore href="{{ page.baseurl }}/get-started/order-tutorial/order-intro.html">Order Processing with REST APIs</a></li>
+        <li><a  data-proofer-ignore href="{{ page.baseurl }}/rest/tutorials/orders/order-intro.html">Order Processing with REST</a></li>
```

Comment:
Consider dropping "... with REST" from the title

Reply:
Because we now have a REST tutorial index page, replaced the two links with a link to that index page.